### DISCUSSION TITLE
括弧類・句読点等の一般的な配置方法

### DISCUSSION THREAD

**Discussion by kidayasuo** [w3c/jlreq-d#60] [jlreq-d-drafts]
[3.4.1 括弧類・句読点等の一般的な配置方法](https://github.com/w3c/jlreq-d/blob/5a5fbfeed7d426234d2ca22ce5ff9c21abb589ff/drafts/3_行の組版処理.md#341括弧類句読点等の一般的な配置方法)の議論

**Comment by kidayasuo** [comment]
@KobayashiToshi 

> 括弧類や句読点は，文章の区切りを示すものである．したがって，括弧類の外側や句読点の後ろに，その区切りを示すために適当な空白を確保する必要がある．

最初に読んだとき、例えば開き括弧は文章の区切りだろうか？　と疑問を持ちました。単語の意味を補足したり、ルビの代わりに読みを示したり、そのような場合、それは文章の区切りかと言われるとハテナな気がします。ただ、本文の区切りでなくても、挿入物にとっては開始、終了であって、これはその意味では区切りでしょうか、そういう意味では区切りなのか、と思い直しました。が、ハテナと思わせない説明になると素晴らしいと思います。「文における何らかの区切り」と「文章の」区切りだとは言わないとか。

また、「したがって……必要がある」は、当然にそれが言えるのかは根拠に少し不安があります。例えば欧文括弧は通常は空白文字を挿入して使いますが、そうでない場合もあります。空白を区切りを示す一手段として捉えて、「区切りを示すものであって、そのためにその区切りに適切な空白を確保する」のように因果を一段緩くして説明するのはどうでしょう。

> また，読者にとっても，慣れた方法が望ましいので，これまで一般的に行われている方法によるのがよいであろう．

これは必要？

> 文字の外枠が正方形である文字を使用している場合は，通常は括弧類や句読点の字幅，つまり文字サイズを字幅として配置しいけばよい．その結果，字形のデザインにもよるが，括弧類の外側又は句読点の後ろに文字サイズの1/2程度の空白が確保される．

これはボディが正方形の場合に限らず、あらゆる場合にそれが言えます。タイプフェースがそのように作られているからです。全てのフォントに言えることとして「これらの文字にはあらかじめ必要な空白が組み込まれているので、通常は字幅通りに配置すれば良い」で良いのではないでしょうか。

一点、和文約物と欧文約物の問題は、混植のところで触れますが、この場所で、和文の中では基本的に和文約物を用いる、といったことに触れた方が良いでしょうか。どうでしょう？　和文約物でない場合にはアキを開けるのに空白文字を使いますので、空白は組み込まれていません。

> 等幅のフォントを変形した場合も，これにならって配置すればよい．
> 注　文字を変形する場合，句点（“。”（U+3002））も変形することになる．この場合，丸の形も変形されるので，この句点だけは除外するという方法もある．
> プロポーショナルな文字の場合は，そのフォントで設定されている字幅で配置していけばよい．

デジタルでは通常その幅のためにデザインされたフォントを用いますので、上のように述べれば、それで十分ではないでしょうか。

文字に変形をかけることのできるアプリケーションもありますが、最初から長体や扁平体に最適化してデザインしたものに対して劣ります。jlreq-dでは変形は扱わなくて良いのではないでしょうか。

（幅が可変のバリアブルフォントは、等倍変形をするのではなく、複数の幅に対して最適化されたグリフがあり、その間を補完して連続的な変化を可能にしています）

> ただし，前述したように，見出し等では，括弧類や句読点の前後の空白について，やや詰める方法や，パーレン（小括弧）などでは，文字サイズの1/2ほど詰める配置方法をとる方法も行われている．

まず本文を基本として説明し、見出し、大きな文字、小さな文字、などの場合は、その章で述べるのはどうでしょう？　なぜかというと、詰める方法は見出しや大きな文字の際には通常の文字でも行われるので、そちらの方で説明した方が背景含めて統一的に説明できるかなと。

**Comment by himorin** [comment]
closing per https://github.com/w3c/jlreq-d/pull/132
check current draft at https://github.com/w3c/jlreq-d/blob/gh-pages/drafts/%E5%AE%9F%E8%A3%85%E7%B7%A8%EF%BC%9A%E8%A1%8C%E3%81%B8%E3%81%AE%E6%96%87%E5%AD%97%E3%81%AE%E9%85%8D%E7%BD%AE.md#33%E6%8B%AC%E5%BC%A7%E9%A1%9E%E5%8F%A5%E8%AA%AD%E7%82%B9%E7%AD%89%E3%81%AE%E9%85%8D%E7%BD%AE
